types/index.d.ts

```diff
@@ -15,11 +15,9 @@ declare global {
         }
 
         interface ChaiHttpRequest {
-            (server: any): ChaiHttp.Agent;
-
-            agent(server: any): ChaiHttp.Agent;
-
-            addPromises(promiseConstructor: PromiseConstructorLike): void;
+            agent?(server: any): ChaiHttp.Agent;
```

Comment:
can you explain why they became optional?

Reply:
WebStorm is showing a warning there, but it seems to be a false-positive thing, so I made all of them required, thanks